stepactions/bundles/pick-cluster-version/0.1/README.MD

```diff
@@ -0,0 +1,15 @@
+# pick-cluster-version stepaction
+
+This StepAction selects a cluster version for provisioning. It determines the target OCP version of an FBC fragment. 
+The version is returned if it is included in the list of supported Hypershift cluster versions.
+
+## Parameters
+|name|description|default value|required|
+|---|---|---|---|
+|FBC_FRAGMENT|A FBC fragment image.||true|
+|CLUSTER_VERSIONS|List of supported minor versions from newest to oldest. E.g. ["4.15","4.14","4.13"]||true|
```

Comment:
Just a question b/c I'm curious and not yet familiar with integration pipelines... Will `CLUSTER_VERSIONS` be specified in the task or pipeline definition by the team implementing this `stepAction` ? Or will it get fed list of versions in some automated way?

Reply:
The pick-cluster-version step is part of the deploy-fbc Pipeline. The preceding step will invoke the [eaas-get-supported-ephemeral-cluster-versions step](https://github.com/konflux-ci/build-definitions/tree/main/stepactions/eaas-get-supported-ephemeral-cluster-versions/0.1) to retrieve the list of Hypershift-supported versions. The output from eaas-get-supported-ephemeral-cluster-versions will then be passed as input to the pick-cluster-version step. I'll add an Example Usage section and it should clarify this.